--- /Applications/LIWC2007/Dictionaries/LIWC2007_English100131.dic	2010-01-30 23:49:27.000000000 -0500
+++ LIWC2007_English100131.dic	2014-04-02 11:28:04.000000000 -0400
@@ -2193,7 +2193,7 @@
 kill*	125	127	129	360
 kilometer*	252	250
 kin	121	122
-kind	<of>131/125	<of>135/126
+kind	131	125	135	126
 kinda	131	135
 kinderg*	354
 kindly	125	126
@@ -2285,7 +2285,7 @@
 lies	125	127	129
 life	146	148
 light	140
-like	(02 134)125/464	(02 134)126	253
+like	2	134	125	464	126	253
 likeab*	125	126
 liked	11	13	125	126
 likel*	131	135
